dynamo-lib/src/cmr/dynamo/connection.clj

```diff
@@ -1,35 +1,89 @@
 (ns cmr.dynamo.connection
   "Contains functions for interacting with the DynamoDB storage instance"
-  (:require [cmr.common.config :as cfg :refer [defconfig]]
+  (:require [clojure.string :as str]
+            [cmr.common.config :as cfg :refer [defconfig]]
+            [cmr.common.services.errors :as errors]
+            [cmr.common.log :refer [debug error info trace warn]]
             [taoensso.faraday :as far]
             [cmr.dynamo.config :as dynamo-config]))
 
+(defn concept-revision-tuple->key
+  [tuple]
```

Comment:
is this how your sending in concept id and revision id?

Reply:
In the get-concepts function in metadata-db/data/oracle/concepts.clj, it has a list of tuples of the form (concept-id, revision-id) that it uses to query the DB. For DynamoDB, I am using that function to parse the tuple so that the values can be used to query DynamoDB

Reply:
yes it does, CMR is never consistent.

Reply:
I hate expanding variables, but maybe you should in this case to avoid the nth code